docs/development/extensions-core/k8s-jobs.md

```diff
@@ -634,25 +653,27 @@ All three examples below are equivalent.
 In all the above cases, Druid will match the selector to any value of task type. Druid applies similar logic for `dataSource`. For `context.tags` setting `null` or an empty object `{}` is equivalent. 
```

Comment:
what are these changes about?

Reply:
Not sure why it is highlighted in red. Doing `git diff` shows no changes done in this line (as well as the Properties header3) below.